### ISSUE TITLE
Ability to disable or re-order bones.

### ISSUE THREAD

**Issue by Dustination** [Ploaj/SSBHLib#103]
Hey, super impressed with this program. It would be great to be able to disable or re-order bones in the hierarchy for easier viewing of specific elements of an animation. If this is outside of the intent of this project, the ability to just disable/enable the 'Trans' bone at the root of the model would be extremely helpful for viewing animations that have baked in motion. If deleting bones is already possible, I was too stupid to figure it out.

Thanks for your consideration.

**Comment by ScanMountGoat** [comment]
By disabling bones do you mean disabling the effect of the animation on a specific bone like "ArmL"? I'm not sure what you mean by reordering bones.

**Comment by Dustination** [comment]
That's correct. This is possible in StudioSB, (inadvertently, I imagine), for example, just by renaming the bone.
